js/holmes.js

```diff
@@ -23,7 +23,7 @@
     /* $FlowIssue - doesn't work with umd */
     root.holmes = factory(document);
   }
-})(this, function (document) {
+})(this, (function (document) {
```

Comment:
After seeing some numbers on this, I still see positive and negative points (see https://twitter.com/getify/status/777945853659152384), but overall it seems better to include this.